skrf/network.py

```diff
@@ -539,6 +539,34 @@ def __pow__(self, other: 'Network') -> 'Network':
         else:
             return cascade(self, other)
 
+    def __rshift__(self, other: 'Network') -> 'Network':
+        """
+        Cascade two 4-port networks with "1=>2/3=>4" port numbering.
+
+        Returns
+        -------
+        ntw : :class:`Network`
+            Cascaded Network
+
+        See Also
+        --------
+        cascade
+
+        """
+        check_nports_equal(self, other)
+        check_frequency_exist(self)
+        (h,_) = shape(self.s[0])
+        if h != 4:
```

Comment:
Before continuing, couldn't it be extended to 2N Networks? (N>1)

Reply:
Also, some additional tests will be needed.

Reply:
Extended to 2N, as requested.

What did you have in mind for testing?
I have a simple Jupyter notebook that I use, but I imagine you're looking for something more amenable to an automated CI/CD flow?

Reply:
> Extended to 2N, as requested.

great, thank you. 
 
> What did you have in mind for testing? I have a simple Jupyter notebook that I use, but I imagine you're looking for something more amenable to an automated CI/CD flow?

Where we are talking about testing, we mean the python located in the `tests` subdirectories. For example, for testing the Network class, there the file `tests/test_network.py`. Theses tests are run automatically for each PR, so it allows to check if nothing is broken and it is work as planned. You will find more documentation here https://scikit-rf.readthedocs.io/en/latest/contributing/index.html#tests. 

Have a look to the tests related to connecting Networks and please do something similar for this new >> operator.

Reply:
(note that the automatic tests performed by github are not working automatically on this PR because this is your first PR... Thats's something imposed by github and we need to run manually the CI tests here in this case...)

Reply:
> Have a look to the tests related to connecting Networks and please do something similar for this new >> operator.

Could you point me to the current test for the `**` operator?
I'm having trouble finding it.
Thanks!

Reply:
It's in `skrf/tests/test_network.py`.

Setup of test networks:
https://github.com/scikit-rf/scikit-rf/blob/790a332ed6269b5067647e90bfad308edff312a6/skrf/tests/test_network.py#L45-L62

Actual test:
https://github.com/scikit-rf/scikit-rf/blob/790a332ed6269b5067647e90bfad308edff312a6/skrf/tests/test_network.py#L342-L344

Reply:
Maybe you can add another function, e.g. `test_cascade_4port()`

Reply:
It's in `skrf/tests/test_network.py`.

Setup of test networks:
https://github.com/scikit-rf/scikit-rf/blob/790a332ed6269b5067647e90bfad308edff312a6/skrf/tests/test_network.py#L45-L62

Actual test:
https://github.com/scikit-rf/scikit-rf/blob/790a332ed6269b5067647e90bfad308edff312a6/skrf/tests/test_network.py#L342-L344

Maybe you can add another test function for this, e.g. `test_cascade_4port()`

Reply:
```
capnf@DESKTOP-G84ND7C MINGW64 ~/Documents/GitHub/scikit-rf (concat-alt)
$ pytest -v -c '""' skrf/tests/test_network.py -k "test_cascade"
============================= test session starts =============================
platform win32 -- Python 3.9.12, pytest-7.1.2, pluggy-1.0.0 -- C:\Users\capnf\anaconda3\envs\skrf-dev\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\capnf\Documents\GitHub\scikit-rf, configfile: ""
plugins: anyio-3.5.0
collecting ... collected 92 items / 90 deselected / 2 selected

skrf/tests/test_network.py::NetworkTestCase::test_cascade PASSED         [ 50%]
skrf/tests/test_network.py::NetworkTestCase::test_cascade2 PASSED        [100%]
```